circle-progressbar/build.gradle

```diff
@@ -0,0 +1,2 @@
+configurations.maybeCreate("default")
+artifacts.add("default", file('circle-progressbar-debug.aar'))
```

Comment:
Is this library not available on any remote repository?

Reply:
This specific library is available at https://github.com/emre1512/CircleProgressBar and can be included via `implementation 'com.emredavarci:circleprogressbar:1.0.4'` but it has a bug that I fixed since the library wasn't working with the bug. 
While it is pending approval I think we can have it added as an *aar* file.

Reply:
Cool.